### PR TITLE
Debian-8: pulsar_rest_api upgrade

### PR DESCRIPTION
**Pull request by kris-lab**

### CODE REVIEW

**Comment by kris-lab** [comment]
@ppp0 @njam please review

File: modules/pulsar_rest_api/manifests/init.pp [hunk 1]
```diff
@@ -7,6 +7,7 @@
   $mongodb_db = 'pulsar-rest-api',
 
   $log_dir = '/var/log/pulsar-rest-api',
+  $config_file = '/etc/pulsar-rest-api/config.yml',
```
Comment (ppp0) [inline modules/pulsar_rest_api/manifests/init.pp]: Is this variable going to be used anywhere else than in line 44?

File: modules/pulsar_rest_api/manifests/init.pp [hunk 2]
```diff
@@ -72,20 +73,19 @@
...
   }
 
-  sysvinit::script { 'pulsar-rest-api':
-    content => template("${module_name}/init.sh"),
-    require => Package['pulsar-rest-api'],
+  daemon { 'pulsar-rest-api':
+    binary  => '/usr/bin/node',
+    args    => "/usr/bin/pulsar-rest-api -c ${config_file}",
+    user    => 'pulsar-rest-api',
+    require => [Package['pulsar-rest-api'], Exec['pulsar-rest-api bower install']],
```
Comment (ppp0) [inline modules/pulsar_rest_api/manifests/init.pp]: `require => User['pulsar-rest-api']` too?

Comment (kris-lab) [inline modules/pulsar_rest_api/manifests/init.pp]: yep, makes sense!

**Comment by ppp0** [comment]
lgtm

**Pull request merged**